.travis.yml

```diff
@@ -15,8 +15,8 @@ install:
   - if [ `uname` = "Linux" ]; then bash setup_travis_linux.sh; fi
   - if [ `uname` = "Linux" ]; then export PATH="/home/linuxbrew/.linuxbrew/bin:$PATH"; fi
   - if [ `uname` = "Linux" ]; then brew install pkg-config; fi
-  - brew tap homebrew/science
-  # Install homebrew/cutest as tap.
+  - brew tap optimizers/cutest
```

Comment:
This was erroneous. I fixed it in https://github.com/optimizers/homebrew-cutest/pull/35/commits/fde69b1f8b21fd4cb73f8e56634493cee1030776.